staff/marta-cervera/challenges/Tic-tac-toe/style.css

```diff
@@ -0,0 +1,29 @@
+body {
+background: url(tic\ tac\ toe.jpg);
```

Comment:
do never use filenames with empty spaces. rename the file to tic-tac-toe.jpg, instead, and update this reference in the css code.